client/components/app-promo/docs/example.jsx

```diff
@@ -0,0 +1,7 @@
+import AppPromo from '..';
+
+export default function AppPromoExample() {
+	return <AppPromo />;
+}
+
+AppPromoExample.displayName = 'AppPromoExample';
```

Comment:
Shouldn't be necessary if we use `AppPromoExample`. Did you mean setting this to:

```suggestion
AppPromoExample.displayName = 'AppPromo';
```